README.md

```diff
@@ -0,0 +1,197 @@
+# pysdccc
+
+This python packages provides a convenient way to execute the [sdccc test suite](https://github.com/Draegerwerk/sdccc/).
+
+This wrapper is only compatible with sdccc versions later than [ebe0e09](https://github.com/Draegerwerk/SDCcc/commit/ebe0e094ff92649d0bda1988b0d1c1b08403aea4).
+
+## Installation
+
+Download from pypi using `pip install pysdccc`
+
+### Development
+
+For this open source project the [Contributor License Agreement](Contributor_License_Agreement.md) governs all relevant activities and your contributions. By contributing to the project you agree to be bound by this Agreement and to licence your work accordingly.
+
+1. clone the repository
+2. install dependencies, e.g. with [`uv sync --dev --all-extras`](https://docs.astral.sh/uv/reference/cli/#uv-sync)
+
+## Usage
+
+### Basic usage
```

Comment:
I know this is the basic usage but short description and a doc string for both main functions would be very helpful.

Reply:
renamed to `Quick start`